### PR TITLE
repositories: Add trojan overlay

### PR DESCRIPTION
**Pull request by YangMame**

### CODE REVIEW

**Comment by YangMame** [comment]
this repo is not belong to me... and I've asked him to register an account

**Comment by mgorny** [comment]
I'm confused now. Do you have permission to add the other overlay?

Also, please `git rebase -i origin` and squash this into two commits, with proper commit messages for each repo added.

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -1953,6 +1953,16 @@ FIN
...
     <feed>https://github.com/microcai/gentoo-zh/commits/master.atom</feed>
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>gig</name>
+    <description>Gig OS's Overlay</description>
+    <homepage>https://github.com/Gig-OS/gig</homepage>
+    <owner type="person">
+      <email>yangmame@i2pmail.org</email>
+    </owner>
+    <source type="git">git://github.com/Gig-OS/gig.git</source>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: We don't use `git://`, only `https://` [+ ssh]. Please follow the example of other repos.

File: files/overlays/repositories.xml [hunk 2]
```diff
@@ -4851,6 +4861,17 @@ FIN
     <feed>https://cgit.gentoo.org/dev/TomWij.git/atom/</feed>
     <!-- <feed>https://cgit.gentoo.org/dev/TomWij.git/rss/</feed> -->
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>trojan-gentoo</name>
+    <description lang="en">Gentoo overlay for trojan.</description>
+    <homepage>https://github.com/trojan-gfw/trojan-gentoo.git/</homepage>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: GitHub sites go without `.git`.

**Comment by YangMame** [comment]
Oh...sorry for that...

**Comment by mgorny** [comment]
Fixed and merged. Thanks.

**Pull request closed without merging**